Sources/SmileID/Classes/Networking/Models/SmartSelfieRequest.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+struct SmartSelfieRequestImage: Encodable {
```

Comment:
This is a generic form data class and not specific to smartselfie. On Android, the Retrofit equivalent is called `MultipartBody.Part`. We should adopt a similar name. It should also be in its own file

Reply:
Fixed